pynamodb/tests/test_model.py

```diff
@@ -3305,6 +3305,33 @@ def test_index_queries(self):
             }
             self.assertEqual(req.call_args[0][1], params)
 
+        with patch(PATCH_METHOD) as req:
+            # equal to:
+            # for item in IndexedModel.email_index.query('foo', IndexedModel.numbers == [1, 2], limit=1):
+            for item in IndexedModel.email_index.get('foo', range_key=[1, 2]):
+                pass
```

Comment:
Can you please explain what happens here? why the for loop is empty? should we be checking the returned items?

Reply:
Nearby tests implement it like:
```
queried = []
for item in IndexedModel.email_index.query('foo', limit=2, user_name__begins_with='bar'):
    queried.append(item._serialize())
```
And then `queried` is never checked, so I decided not to create stub array in this case.

Anyway my implementation of .get() was wrong, as it should return single item, not iterable. Please check updated PR. Thanks.